src/Grpc/Grpc.csproj

```
@@ -18,7 +18,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Protobuf Include="$(EngRoot)proto/protos/orchestrator_service.proto" GrpcServices="both" Link="proto/orchestrator_service.proto" />
+    <Protobuf Include="*.proto" GrpcServices="Both" />
```

Consider a recursive glob just in-case we structure this folder down the road.

```suggestion
    <Protobuf Include="**/*.proto" GrpcServices="Both" />
```

---

`*/*.proto` didn't work, but I got it working with `**/*.proto`.